posenet/src/multiPose/decodeMultiplePoses.ts

```diff
@@ -114,13 +114,14 @@ const kLocalMaximumRadius = 1;
  * @return An array of poses and their scores, each containing keypoints and
  * the corresponding keypoint scores.
  */
-export default async function decodeMultiplePoses(
+export async function decodeMultiplePoses(
     heatmapScores: tf.Tensor3D, offsets: tf.Tensor3D,
     displacementsFwd: tf.Tensor3D, displacementsBwd: tf.Tensor3D,
     outputStride: number, maxPoseDetections: number, scoreThreshold = 0.5,
     nmsRadius = 20): Promise<Pose[]> {
   const poses: Pose[] = [];
 
+  // tslint:disable-next-line:max-line-length
```

Comment:
I wasn't sure what to do here, as when this line isn't commented, out I get a max-line length error, but when I change it to be something like:

```javascript
const [
  scoresBuffer, 
  offsetsBuffer,
  displacementsFwdBuffer, 
  displacementsBwdBuffer
] = 
```
and then save, clang formatter collapses it to one line.